src/controller/python/chip/ChipDeviceCtrl.py

```diff
@@ -371,6 +381,40 @@ def attestationChallenge(self) -> bytes:
 
         return bytes(buf)
 
+    @property
+    def sessionAllowsLargePayload(self) -> bool:
+        self._dmLib.pychip_SessionAllowsLargePayload.argtypes = [ctypes.c_void_p, POINTER(ctypes.c_bool)]
+        self._dmLib.pychip_SessionAllowsLargePayload.restype = PyChipError
+
+        supportsLargePayload = ctypes.c_bool(False)
+
+        builtins.chipStack.Call(
+            lambda: self._dmLib.pychip_SessionAllowsLargePayload(self._deviceProxy, pointer(supportsLargePayload))
+        ).raise_on_error()
+
+        return supportsLargePayload.value
+
+    @property
+    def isSessionActive(self) -> bool:
```

Comment:
Here too, and also the naming mentioned above.
```suggestion
    def IsActiveSession(self) -> bool:
```